ads/aja.js

```diff
@@ -28,6 +28,8 @@ export function aja(global, data) {
     sspCode: data['sspCode'],
   });
 
+  const elAttri = 'allow-same-origin allow-top-navigation ';
+  const elAttri2 = 'allow-scripts allow-popups ';
```

Comment:
Are you sure you want to add all of these? They will only apply to the inner iframe, not the top level iframe. Is that OK?

Reply:
Thank you very much. Maybe it's okay, is there a problem?